sample-code/ic-customized-workflow-app/IC_customized_workflow_using_cc_storage.sh

```diff
@@ -44,8 +44,7 @@ curl -s $ic_endpoint \
     "href": "'$intermediate_output_url'",
     "mask": {
       "format":"soft"
-    },
-    "overwrite":true
```

Comment:
@khound we should leave the flag in for CC Storage examples

Reply:
Yes. I can put "overwrite":<overwrite_if_applicable> here